src/main/resources/locale_pt_PT.properties

```
@@ -0,0 +1,196 @@
+command.roles.sub.managerRole.set=A função de gestor definida como %ROLE%.
+command.roles.sub.remove=Remover uma função de reputação.
+command.roles.sub.remove.notARepRole=Esta função não é uma função de reputação.
+command.roles.sub.remove.removed=Remover um papel %ROLE%.
+command.scan.canceled=Scan cancelado.
+command.scan.canceling=Tentativa de cancelar o scan.
+command.scan.completed=Varredura concluída.
+command.scan.description=Scan a channel for reputations.
+command.scan.error.history=Desaparecimento da permissão para histórico de mensagens
+command.scan.error.queueFull=Todos os scans estão actualmente em execução. Por favor aguarde até que uma ranhura esteja livre.
+command.scan.error.running=Já está a decorrer uma tarefa nesta guild. Por favor aguarde até que a tarefa esteja terminada.
+command.scan.progress=Progress: %PERCENT%
+command.scan.result=Scanned %SCANNED% messages.\nAdded %HITS% reputation.
+command.scan.scheduling=Agramação do seu scan. O tempo necessário estimado até à conclusão é: %DURATION%
+command.scan.sub.cancel=Cancelar um scan em execução.
+command.scan.sub.cancel.noTask= Nenhuma tarefa está em execução para esta guilda.
+command.setup.description=Executar uma configuração básica do bot.
+command.setup.dialog.channels=Por favor introduza todos os canais onde os utilizadores têm permissão para receber reputação.
+command.setup.dialog.continueToProceed=Please press "$word.continue$" to proceed.
+command.setup.dialog.loadDefaults=Carregar palavras de agradecimento por defeito para as suas línguas.
+command.setup.dialog.managerRole=Please write one role which should be allowed to change the settings of the bot.
+command.setup.dialog.roles=Por favor defina alguns papéis de reputação.\nEstes papéis serão dados a um utilizador quando este atingir a quantidade especificada de reputação.\nDefina-os como <$words.role$> <$words.reputation$>.\nQuando tiver definido todos os seus papéis prima o botão.
+command.setup.dialog.rolesFormat=Por favor defina-os como <$words.role$> <$words.reputation$>
+command.setup.dialog.selectLanguage=Please select your language.
+command.setup.dialog.setupComplete=Setup complete!
+command.setup.dialog.starting=Arrancar setup.
+command.setup.dialog.welcome=Bem-vindos à configuração do bot de reputação.
+command.thankwords.description=Configurar palavras.de.agradecimento
+command.thankwords.error.patternNotFound=Pattern não existe.
+command.thankwords.sub.add=Add another pattern. Padrões são interpretados como Regex. Um padrão não pode conter espaços.
+command.thankwords.sub.add.added=Adicionar padrão %REGEX%.
+command.thankwords.sub.check=Cheque se uma mensagem daria reputação.
+command.thankwords.sub.check.confidence=Confidence Score: %SCORE%
+command.thankwords.sub.check.match.answer=Aresposta Match
+command.thankwords.sub.check.match.fuzzy=Fuzzy Match
+command.thankwords.sub.check.match.mention=Mention Match
+command.thankwords.sub.check.match.noMatch=No Match
```

Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.check.match.noMatch=Sem correspondência
```